CONTRIBUTING.md

```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+## Bug fixes
+
+Issues can be found on our [GitHub issues](https://github.com/yummygum/react-flagpack/issues) page. Found a bug? Let us know! Report the bug by [creating a new issue](https://github.com/Yummygum/react-flagpack/issues/new/choose). Make sure to check the issues page before creating a new bug issue to prevent duplicate issues. Once you start fixing a bug, post your intent on the issue to prevent others diving in at the same time.
+
+## New features
+
+Upcoming new features can be found on our [GitHub issues](https://github.com/yummygum/react-flagpack/issues) page. Do you want to contribute by adding new features, open a [Feature Request](https://github.com/Yummygum/react-flagpack/issues/new/choose).
+
+## Documentation
+
+If a PR introduces or changes API, mention one of the Yummygum team members to make sure it'll be processed into the [website docs](https://flagpack.xyz/docs/).
+
+## Development
+
+Follow these steps to get your code PR-ready:
+
+1. Fork the `react-flagpack` repo (and also the `flagpack-core` if you want to make changes to the core)
```

Comment:
Let's, somehwere in this document, reference the `flagpack-core` repo with a link.